CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [[Unreleased]]
 - Included `ctid` field in the `tx` request.
 
+### Added
+- Support for the PriceOracle amendment (XLS-47d).
```

Comment:
```suggestion
- Support for the Price Oracles amendment (XLS-47d).
```
nit

Reply:
I feel the version number of the XLS spec is useful, if someone needs to review this implementation in the future.
But if you feel strongly about it, I can remove it

Reply:
I recommended to remove the letter from its postfix because we've done this for other amendments in CHANGELOG.